ui/src/hosts/components/HostsTable.js

```diff
@@ -123,45 +121,52 @@ const HostsTable = React.createClass({
         </div>
         <div className="panel-body">
-          <table className="table v-center">
-            <thead>
-              <tr>
-                <th
-                  onClick={() => this.updateSort('name')}
-                  className={this.sortableClasses('name')}
-                >
-                  Host
-                </th>
-                <th
-                  onClick={() => this.updateSort('deltaUptime')}
-                  className={this.sortableClasses('deltaUptime')}
-                  style={{width: '74px'}}
-                >
-                  Status
-                </th>
-                <th
-                  onClick={() => this.updateSort('cpu')}
-                  className={this.sortableClasses('cpu')}
-                  style={{width: '70px'}}
-                >
-                  CPU
-                </th>
-                <th
-                  onClick={() => this.updateSort('load')}
-                  className={this.sortableClasses('load')}
-                  style={{width: '68px'}}
-                >
-                  Load
-                </th>
-                <th>Apps</th>
-              </tr>
-            </thead>
-            <tbody>
-              {sortedHosts.map(h => {
-                return <HostRow key={h.name} host={h} source={source} />
-              })}
-            </tbody>
-          </table>
+          {hostCount > 0 && !hostsError.length
+            ? <table className="table v-center table-highlight">
+                <thead>
+                  <tr>
+                    <th
+                      onClick={() => this.updateSort('name')}
+                      className={this.sortableClasses('name')}
+                    >
+                      Host
+                    </th>
+                    <th
+                      onClick={() => this.updateSort('deltaUptime')}
+                      className={this.sortableClasses('deltaUptime')}
+                      style={{width: '74px'}}
+                    >
+                      Status
+                    </th>
+                    <th
+                      onClick={() => this.updateSort('cpu')}
+                      className={this.sortableClasses('cpu')}
+                      style={{width: '70px'}}
+                    >
+                      CPU
+                    </th>
+                    <th
+                      onClick={() => this.updateSort('load')}
+                      className={this.sortableClasses('load')}
+                      style={{width: '68px'}}
+                    >
+                      Load
+                    </th>
+                    <th>Apps</th>
+                  </tr>
+                </thead>
+
+                <tbody>
+                  {sortedHosts.map(h => {
+                    return <HostRow key={h.name} host={h} source={source} />
+                  })}
+                </tbody>
+              </table>
+            : <div className="generic-empty-state">
```

Comment:
Not sure if how I did this was a worthwhile change. Goal was to have a more visually consistent empty state as the rest of the UI, instead of just a blank table. Will likely revisit in a different (aka smaller) PR